about/about.html

```diff
@@ -0,0 +1,96 @@
+                        <li><a href="#news" class="nav-list">News</a></li>
+                        <li><a href="#campain" class="campain">CC Campain</a></li>
+                    </ul>
+                </div>
+            </nav>
+
+            <div class="title">
+                <p class="greeting"><strong>Hello World!</strong></p>
+                <h1>Artifitial Intelligence Webinar 2022</h1>
+            </div>
+            <div class="disc">
+                <p>ERP solution for freight forwarders to overcome technology barriers, manage their back office
+                    operations more
+                    efficiently so that they can improve their per-capita productivity and per-shipment profitability.
+                    The life we are living today and how is the Artificial intelligence is evolving to
+                    our daily life the advantages and disadvanteges of it and how to and how to hundle the side-effect
+                    of AI.
+                </p>
+            </div>
+            <div class="time">
+                <p>please reach-out to us throught the email for further information
+                    <br>Artificial Intelligence Webinar 2022!<br>
+                    <strong>aiwebinar2022@technolog.org</strong>
+                </p>
+            </div>
+        </div>
+    </header>
+
+    <section class="logo">
+        <h3>Creative Amazing <br> Artificial Intelligence Webinar 2022 Logo</h3>
+        <a href="./index.html" class="logo">
+            <img src="imagesabout/Aii.jpg" alt="logo"></a>
+    </section>
+
+    <section class="past">
+
+        <h4>See the past Artificial intelligence webinars</h4>
+        <p> Take look at two Artificial intelligence webinars happened in the past on zoom </p>
+        <div class="sample">
+            <div class="Event">
+                <div class="color">
+                    <h5>2020</h5>
+                    <p><strong>Artificial intelligence webinars Zoom hosting</strong></p>
+                </div>
+            </div>
+
+            <div class="events">
+                <div class="colora">
+                    <h5>2021</h5>
+                    <p><strong>Artificial intelligence webinars Google-meet</strong></p>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="sponsors">
+        <h3>Sponsors</h3>
+        <div class="images">
+            <img src="imagesabout/airbnb.svg" alt="airbnb">
+            <img src="imagesabout/google.svg" alt="google">
+            <img src="imagesabout/amazon-png.png" alt="amazon">
+            <img src="imagesabout/facebook.svg" alt="facebook">
+            <img src="imagesabout/mozilla.svg" alt="mozilla">
+        </div>
+    </section>
+    <footer class="refera">
+        <a href="./index.html" class="menu">
+            <img src="./imagesabout/logo.jpg" alt="logo"></a>
+        <p>2022 Artificial intelligence Webinar templete was disigned by Cindy Shin.</p>
+    </footer>
```

Comment:
- [ ] The footer on the About Page is meant to have a dark background as seen in the original design

![image](https://user-images.githubusercontent.com/4247740/159138390-e674c2ab-d1f8-425c-bd4f-76e98d0cd5d7.png)

Reply:
resolved 👍